MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -301,6 +329,21 @@ public bool SupportsFilterWithOption(FilterOptionType option)
             return false;
         }
 
+        public bool IsVaapiDeviceAmd()
+        {
+            return _isVaapiDeviceAmd;
+        }
```

Comment:
You could also do private set
```suggestion
        public bool IsVaapiDeviceAmd { get; private set; }
```